### ISSUE TITLE
Feature Request: Combine multiple years of database stats

### ISSUE THREAD

**Issue by chunter2** [jeffpiazza/derbynet#321]
I haven't fully thought this through but now that I've got multiple years of data I was wondering if there would be a way to combine standings to see the fastest overall.  I've used the same instance of DerbyNet for different tracks/timers so I don't think it would make sense to combine all databases.  Just starting the discussion.  :)

Thanks

**Comment by jonathanofsmith** [comment]
@chunter2 - have you checked out any free/open source reporting tools as an alternative?
Excel is obviously the most well-known. But there are also tools like Tableau Public, etc. which would allow you to export from DerbyNet and upload the result sets, then analyze them.

**Comment by chunter2** [comment]
Yes, I've used Power BI for that kind of stuff.  It was more just a thought exercise for something in DerbyNet itself.  It probably doesn't make sense unless all the data was in one database which isn't the design.
